src/main/java/com/quantumlytangled/gravekeeper/util/SoulboundHandler.java

```diff
@@ -25,9 +25,10 @@ public static boolean isSoulbound(@Nonnull final ItemStack itemStack) {
     // check for NBT tags
     final NBTTagCompound tagCompound = itemStack.getTagCompound();
     if (tagCompound != null) {
-      for (final String tagBoolean : GraveKeeperConfig.SOULBOUND_TAG_BOOLEAN) {
-        if (tagCompound.hasKey(tagBoolean)
-            && tagCompound.getBoolean(tagBoolean)) {
+      for (final String tagValue : GraveKeeperConfig.SOULBOUND_TAG_BOOLEAN) {
+        final String[] tagSplit = tagValue.split(":");
```

Comment:
Make a map during config load instead of parsing multiple times during execution.
boolean toString isn't a 0/1, we're not in C/C++. Directly compare boolean to boolean.